graph-manager-docs/source/managed-federation/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Managed federation overview
+sidebar_title: Overview
+---
+
+Apollo Graph Manager provides free **managed federation** features for data graphs that use [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/).
+
+With managed federation, your [gateway](https://www.apollographql.com/docs/apollo-server/federation/gateway/) is no longer responsible for combining your [implementing service](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) schemas into a single federated schema. Instead, your implementing services push their schemas to Graph Manager, which combines them. Your gateway then fetches the federated schema and associated configuration from Graph Manager:
+
+```mermaid
+graph BT;
+  subgraph " "
+  serviceA[Products service];
+  serviceB[Reviews service];
+  gateway([Gateway]);
+  end
+  graphManager{{Apollo Graph Manager}};
+  serviceA & serviceB -. Pushes schema .-> graphManager;
+  gateway -. Regularly polls for combined schema .-> graphManager;
+  class graphManager secondary;
+```
+
+This architecture helps your organization safely validate, coordinate, deploy, and monitor changes to your graph:
```

Comment:
There are a number of reasons why you need a system in place that's separate from live introspection of your services and what managed federation enables. One of the core ones is that you can actually do things that you otherwise couldn't, like migrating fields from one IS to another, adding circular dependencies, etc.

Additionally, it's important to make sure that the gateway is only composing a _stable_ set of SDLs (ones that are fully supported by the underlying services). In the case of a roll-out of your services, that won't be the case, but you have no control over when a gateway is going to recompose if it does it on start-up (servers restart all the time)

Managed federation gives you full control over how your gateway changes over time and allows you the freedom to treat your gateway as infrastructure that changes infrequently, which promotes stability.